src/reader/mod.rs

```diff
@@ -77,6 +77,19 @@ macro_rules! builder_methods {
 
         /// Changes whether mismatched closing tag names should be detected.
         ///
+        /// Note, that start and end tags [should match literally][spec], they cannot
+        /// have different prefixes even if both prefixes resolves to the same namespace.
+        /// The
```

Comment:
Should probably be "The XML ... is not valid, even though sematically the start tag is the same as the end tag."

"The reason is that namespaces _are_..."